src/platform/packages/shared/kbn-esql-validation-autocomplete/src/autocomplete/helper.ts

```diff
@@ -233,10 +232,13 @@ export function getOverlapRange(
     }
   }
 
-  // add one since Monaco columns are 1-based
+  if (overlapLength === 0) {
```

Comment:
Not sure if this is because of this change but something goes wrong with autocomplete in KEEP

![meow](https://github.com/user-attachments/assets/264a5a6d-96d4-47d5-9333-906ea7956bd0)

Same in SORT and maybe other places (this works badly even without FORK). This is a bug with multilines

Reply:
Yeah, probably my changes

Reply:
Fixed in https://github.com/elastic/kibana/pull/216743/commits/3fc6a054126c00fc9d950af711e0a89fb0ca6c5f


https://github.com/user-attachments/assets/5828c95b-37b5-4fb5-bb90-3d5690649ce2

Reply:
Also fixed